ci/test_size_regression.sh

```diff
@@ -2,7 +2,7 @@
 
 # Checks the absolute size and the relative size increase of a file.
 
-MAX_SIZE=5900000 # 5.9MB
+MAX_SIZE=6000000 # 6MB
```

Comment:
Why is this increasing in this PR?

Reply:
I am guessing compilation differences between clang 8 and clang 10? Not entirely sure. We _need_ to run the size analysis again.